dna-diffusion/requirements.yml

```diff
@@ -0,0 +1,29 @@
+name: dnadiffusion
+channels:
+  - conda-forge
+  - bioconda
```

Comment:
could add `nodefaults` to avoid accidentally polluting the environment with anaconda packages